packages/ui-core/src/utils/cn.ts

```diff
@@ -9,7 +9,7 @@ interface IClassSet {
 }
 
 type CustomClassNameType = string;
-type CustomClassNamesArrayType = string[];
+type CustomClassNamesArrayType = Array<string | undefined>;
```

Comment:
Вроде Array - это вроде старая нотация, можем сделать так?
```suggestion
type CustomClassNamesArrayType = (string | undefined)[];
```
Или 
```suggestion
type CustomClassNameType = string | undefined;
type CustomClassNamesArrayType = CustomClassNameType[];
```